src/test/java/com/revature/services/JMeterResponseCollectorTest.java

```diff
@@ -0,0 +1,216 @@
+    private JMeterResponseCollector jmrc;
+    private StandardJMeterEngine engine;
+    private static final String JMeterPropPath = "src/test/resources/test.properties";
+    private SampleEvent event1;
+    private SampleEvent event2;
+    private SampleEvent event3;
+    private SampleEvent event4;
+    private SampleEvent event5;
+    private SampleResult event1Result;
+    private SampleResult event2Result;
+    private SampleResult event3Result;
+    private SampleResult event4Result;
+    private SampleResult event5Result;
+
+    @BeforeAll
+    static void setUpBeforeClass() throws Exception {
+    }
+
+    @AfterAll
+    static void tearDownAfterClass() throws Exception {
+    }
+
+    @BeforeEach
+    void setUp() throws Exception {
+
+        engine = new StandardJMeterEngine();
+        JMeterUtils.loadJMeterProperties(JMeterPropPath);
+        Summariser summer = null;
+     
+        
+        jmrc = new JMeterResponseCollector(summer);
+        
+
+        event1Result = SampleResult.createTestSample(100,200);
+        event1Result.setLatency(100);
+        event1Result.setResponseCode("200");
+        event1 = new SampleEvent(event1Result, "Sample Thread Group");
+        
+        event2Result = SampleResult.createTestSample(200, 400);
+        event2Result.setLatency(200);
+        event2Result.setResponseCode("400");
+        event2 = new SampleEvent(event2Result, "Sample Thread Group");
+        
+        event3Result = SampleResult.createTestSample(400, 800);
+        event3Result.setLatency(400);
+        event3Result.setResponseCode("400");
+        event3 = new SampleEvent(event3Result, "Sample Thread Group");
+        
+        event4Result = SampleResult.createTestSample(800, 1100);
+        event4Result.setLatency(300);
+        event4Result.setResponseCode("400");
+        event4 = new SampleEvent(event4Result, "Sample Thread Group");
+        
+        event5Result = SampleResult.createTestSample(1100, 1350);
+        event5Result.setLatency(250);
+        event5Result.setResponseCode("400");
+        event5 = new SampleEvent(event5Result, "Sample Thread Group");
+        
+
+    }
+
+    @AfterEach
+    void tearDown() throws Exception {
+        jmrc.setFailCount(0);
+        jmrc.setCurrentTime(0);
+        jmrc.setStartTime(0);
+        jmrc.setResponseMax(0);
+        jmrc.setLatencyTimes(null);
+        jmrc.setOkResponse(0);
+    }
+    
+    @Test
+    void testgetsuccessFailPercentageNoEvents() {
+        long expected = (long) 0;
+        assertEquals(expected, jmrc.getsuccessFailPercentage());
+    }
+    
+    @Test
+    void test200ResponseNoEvents() {
+        int expected = 0;
+        assertEquals(expected, jmrc.getOkResponse());
+    }
+    
+    @Test
+    void test400ResponseNoEvents() {
+        int expected = 0;
+        assertEquals(expected, jmrc.getFailCount());
+    }
+    
+    @Test
+    void test200ResponseEvent() {
+        int expected = jmrc.getOkResponse() + 1;
```

Comment:
Not entirely clear why we need to do getOkResponse + 1. Shouldn't getOkResponse always be 0 here?